db/app.js

```diff
@@ -1,20 +1,24 @@
 const express = require("express");
 const { getTopics } = require("../controllers/topics.controller");
-const apiController = require('../controllers/api.controller')
+const { getArticleById} = require('../controllers/articles.controller')
+const {getEndpoints} = require('../controllers/api.controller')
+
 
 const app = express();
 
-app.use("/api/topics", getTopics);
 
 app.use(express.json())
-app.use('/api', apiController.getEndpoints)
+app.get("/api/topics", getTopics);
+
+app.get('/api', getEndpoints)
 
+app.get('/api/articles/:article_id', getArticleById)
 
 app.use((err, req, res, next) => {
   if (err.status && err.msg) {
     res.status(err.status).send({ msg: err.msg });
   } else {
-    res.status(500).send({ msg: "Internal Server Error" });
+    res.status(404).send({ msg: "Not Found" });
```

Comment:
This will need refactoring when you get further into the error handling of the project.